README.md

```diff
@@ -88,6 +88,16 @@ Possible values: `true`, `false`
 
 Compress the resulting GeoJSON or TopoJSON with Geobuf.
 
+#### options.rename
```

Comment:
Please define the default then as `null`, also in the task options sections in the JS file